blocks/products/products.js

```diff
@@ -0,0 +1,79 @@
+import { createNanoBlock, renderNanoBlocks } from '../../scripts/nano-blocks.js';
+
+const FETCH_URL = 'https://www.bitdefender.com/site/Store/ajax';
+
+/**
+ * Fetches a product from the Bitdefender store.
+ * @param code The product code
+ * @param variant The product variant
+ * @returns {Promise<*>}
+ */
+async function fetchProduct(code = 'av', variant = '1u-1y') {
```

Comment:
Should this be outside of products since it might be used from other blocks (like product comparison table) ?